Cargo.toml

```diff
@@ -13,16 +13,16 @@ edition = "2018"
 [badges]
 travis-ci = { repository = "mullvad/pfctl-rs" }
 
-
 [dependencies]
+derive_builder = "0.9"
 errno = "0.2"
-error-chain = "0.12"
 ioctl-sys = "0.6.0"
-libc = "0.2.29"
-derive_builder = "0.9"
 ipnetwork = "0.16"
+libc = "0.2.29"
+thiserror = "1"
 
 [dev-dependencies]
 assert_matches = "1.1.0"
-uuid = { version = "0.8", features = ["v4"] }
+error-chain = "0.12"
 scopeguard = "1.0"
+uuid = { version = "0.8", features = ["v4"] }
```

Comment:
You still move stuff around a lot here. So there are still code changes even if the resulting dependencies are the same. I would appreciate if unrelated code is not changed :pray:

Reply:
It was a simple `cargo sort`